### DISCUSSION TITLE
htmlbeautifier

### DISCUSSION THREAD

**Discussion by olariuromeo** [tabler/tabler#1195] [General]
run bundle instal get this error : htmlbeautifier-1.4.1 requires ruby version >= 2.6.0, which is incompatible with the current version, ruby 2.5.5p157 but [Install Ruby](https://ruby-lang.org/en/documentation/installation/) - the recommended version is [2.5.5](https://cache.ruby-lang.org/pub/ruby/2.5/ruby-2.5.5.tar.gz).

**Comment by dheineman** [comment]
Can you try using ruby 2.7

I am not quite sure why there is a lower version in the readme for osx users but i use  ruby 2.7.4p191 on a Apple m1 which works just fine.
